static/js/commentDataManager.js

```diff
@@ -27,7 +27,7 @@ commentDataManager.prototype.getComments = function() {
 }
 
 commentDataManager.prototype.getRepliesOfComment = function(commentId) {
-  return this.comments[commentId].replies;
+  return this.comments[commentId] && this.comments[commentId].replies;
```

Comment:
É melhor sempre retornar o mesmo tipo de dado. Se `this.comments[commentId]` for _falsy_, vai retornar um booleano. Melhor retornar um `{}` para evitar problemas. É o mesmo caso do que comentei no PR do EIS...

Reply:
boa